tests/Event/DispatcherTest.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+namespace PhpAb\Event;
+
+class DispatcherTest extends \PHPUnit_Framework_TestCase
+{
+    public function testDispatchEventWithoutListeners()
+    {
+        // Arrange
+        $dispatcher = new Dispatcher();
+
+        // Act
+        $result1 = $dispatcher->dispatch('event', null);
+        $result2 = $dispatcher->dispatch('event2', null);
+
+        // Assert
+        $this->assertNull($result1);
```

Comment:
Is this test needed? Dispatch always returns null and you;re checking that in every test again. Maybe you need one test: `testDispatchReturnsNull` and just check if it returns null. Than you don't have to add that assert in all the other tests.